Workspace/src/rover_gui/include/rover_gui/gui.h

```diff
@@ -3,7 +3,9 @@
 #define GUI_H
 
 #define UTM_POSE_TOPIC "/localization/pose_utm"
-#define ARM_POSE_TOPIC "/arm_interface/actual_joint_angles"
+#define ARM_POSE_TOPIC_ACTUAL "/arm_interface/actual_joint_angles"
+#define ARM_POSE_TOPIC_DESIRED "/arm_interface/desired_joint_angles"
```

Comment:
All of these should not be #defines. It should probably be const strings objects

Reply:
should I leave the UTM_POSE_TOPIC or edit the mapwidget to include it as a const string?

Reply:
Change that too

Reply:
Tbh they should be launch file params that have a default value

Reply:
can I edit the autonomy control widget as well? It also depends on these defines

Reply:
Ye. Edit everything u need to

Reply:
Alright got it, now all topics are in a cfg file and I made a launch file